pubsub/gcloud/aio/pubsub/metrics.py

```diff
@@ -0,0 +1,57 @@
+from gcloud.aio.auth import BUILD_GCLOUD_REST
+
+if BUILD_GCLOUD_REST:
+    pass
+else:
+    import prometheus_client
+
+    _NAMESPACE = 'gcloud_aio'
+    _SUBSYSTEM = 'pubsub'
+
+    BATCH_SIZE = prometheus_client.Histogram(
+        'subscriber_batch',
+        'Histogram of number of messages pulled in a single batch',
+        namespace=_NAMESPACE,
+        subsystem=_SUBSYSTEM,
+        unit='size',
+        buckets=(0, 1, 5, 10, 25, 50, 100, 150, 250, 500, 1000, 1500, 2000,
+                 5000, float('inf')))
+
+    CONSUME = prometheus_client.Counter(
+        'subscriber_consume',
+        'Counter of the outcomes of PubSub message consume attempts',
+        ['outcome'],
+        namespace=_NAMESPACE,
+        subsystem=_SUBSYSTEM,
+        unit='total')
```

Comment:
I believe we've discovered that the `unit` flag should not be provided to `Counter` types, as it leads to `<metricname>_total_total` naming.

Reply:
Fair point, I'll update this.